users/tests.py

```diff
@@ -11,13 +11,18 @@
 from plio.cache import get_cache_key
 
 
-class OtpAuthTestCase(BaseTestCase):
+class OtpAuthTestCase(BaseTestCase):  
```

Comment:
There's an extra space character added here
Usually it's best to see the "Files Changed" tab in a pull request yourself so you can see what all got changed from your commits

Reply:
I guess pre commit is also failing. All the spacing issues should be fixed by pre commit

Reply:
done